Sources_v3/APIClient/Endpoints/Requests/EditMessageRequestBody.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct EditMessageRequestBody: Encodable {
+    private enum CodingKeys: String, CodingKey {
+        case clientId = "client_id"
+        case message
+    }
+    
+    let clientId: UserId
+    let text: String
+    
+    func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+
+        try container.encode(clientId, forKey: .clientId)
+        try container.encode(["text": text], forKey: .message)
+    }
+}
```

Comment:
Do we need this? Maybe we could be able to use `MessageRequestBody`, the structure looks similar.